app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt

```diff
@@ -89,7 +90,8 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
         // https://github.com/mozilla-mobile/fenix/issues/14279
         // To prevent GeckoView from resizing we're going to change the softInputMode to not adjust
         // the size of the window.
-        requireActivity().window.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_NOTHING)
+        if (VERSION.SDK_INT < VERSION_CODES.R)
```

Comment:
Is there something that should be done here for >R?

Reply:
Nope. The inset listener set up in the activities will handle that case.